.env_sample

```diff
@@ -40,6 +40,15 @@ export ELASTICSEARCH_URL="http://localhost:9200"
 # SSL config
 export FORCE_SSL_IN_RAILS="not applicable in dev"
 
+# Postgres DB
+# export DATABASE_URL="not applicable in dev. Set by docker-compose."
```

Comment:
This wording is maybe not ideal, not too many people actually use the docker-compose workflow, even on our team.

I'd probably also use actual comments for the explanations and examples for the values, e.g.

```
# Not applicable when using docker-compose
# See: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING
# export DATABASE_URL="postgresql://[user[:password]@][netloc][:port][,...][/dbname][?param1=value1&...]"
```

Maybe we could even set a default that will work for quite a few local installs, e.g.

```
export DATABASE_URL="psql postgres:///Forem_development"
```

Reply:
Yeah good point. These aren't strictly necessary but thought it could be useful to include available config variables.